youtube_dl/extractor/rbmaradio.py

```diff
@@ -13,15 +13,15 @@
 
 
 class RBMARadioIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?rbmaradio\.com/shows/(?P<show_id>[^/]+)/episodes/(?P<id>[^/?#&]+)'
+    _VALID_URL = r'https?://(?:www\.)?redbullradio\.com/shows/(?P<show_id>[^/]+)/episodes/(?P<id>[^/?#&]+)'
```

Comment:
Sorry, unfamiliar with this code.

Fixed, and force pushed a cleaned version of the branch.